core/src/main/java/io/questdb/cairo/sql/PageFrameCursor.java

```diff
@@ -25,13 +25,17 @@
 package io.questdb.cairo.sql;
 
 import io.questdb.cairo.TableReader;
+import io.questdb.std.IntList;
 import io.questdb.std.QuietCloseable;
 import org.jetbrains.annotations.Nullable;
 
 public interface PageFrameCursor extends QuietCloseable, SymbolTableSource {
 
     void calculateSize(RecordCursor.Counter counter);
 
+    // returns local (query) to table reader index mapping
```

Comment:
can we take this opportunity to elaborate why this mapping is needed. It was necessitated by parquet but no explanation given

Reply:
Added a more detailed javadoc in 99216c6373c417ce1d29d94778a1cba08e6f921d